tests/integration/components/dynamic-scan-test.js

```diff
@@ -0,0 +1,968 @@
+          .dom(
+            `[data-test-projectPreference-osVersionSelect] .${classes.trigger}`
+          )
+          .hasText(this.devicePreference.platform_version);
+
+        await selectChoose(
+          `[data-test-projectPreference-deviceTypeSelect] .${classes.trigger}`,
+          `t:${deviceType([ENUMS.DEVICE_TYPE.PHONE_REQUIRED])}:()`
+        );
+
+        // verify ui
+        assert
+          .dom(
+            `[data-test-projectPreference-deviceTypeSelect] .${classes.trigger}`
+          )
+          .hasText(`t:${deviceType([ENUMS.DEVICE_TYPE.PHONE_REQUIRED])}:()`);
+
+        // verify network data
+        assert.strictEqual(
+          this.requestBody.device_type,
+          `${ENUMS.DEVICE_TYPE.PHONE_REQUIRED}`
+        );
+
+        const filteredDevices = this.availableDevices.filter(
+          (it) => it.platform === 0 && !it.is_tablet
+        );
+
+        await selectChoose(
+          `[data-test-projectPreference-osVersionSelect] .${classes.trigger}`,
+          filteredDevices[1].platform_version
+        );
+
+        // verify ui
+        assert
+          .dom(
+            `[data-test-projectPreference-osVersionSelect] .${classes.trigger}`
+          )
+          .hasText(filteredDevices[1].platform_version);
+
+        // verify network data
+        assert.strictEqual(
+          this.requestBody.platform_version,
+          filteredDevices[1].platform_version
+        );
+
+        // enable api catpure
+        await click(
+          '[data-test-dynamicScanModal-runApiScanFormControl] [data-test-dynamicScanModal-runApiScanCheckbox]'
+        );
+
+        // verify api-filter render
+        let apiFilterRows = findAll('[data-test-apiFilter-row]');
+
+        assert.strictEqual(apiFilterRows.length, 2);
+
+        assert
+          .dom(
+            apiFilterRows[0].querySelectorAll('[data-test-apiFilter-cell]')[0]
+          )
+          .hasText('api.example.com');
+
+        assert
+          .dom(
+            apiFilterRows[1].querySelectorAll('[data-test-apiFilter-cell]')[0]
+          )
+          .hasText('api.example2.com');
+
+        if (automatedScan) {
+          assert
+            .dom('[data-test-dynamicScanModal-automatedDynamicScanScheduleBtn]')
+            .isNotDisabled()
+            .hasText('t:scheduleDynamicscan:()');
+
+          await click(
+            '[data-test-dynamicScanModal-automatedDynamicScanScheduleBtn]'
+          );
+        } else {
+          assert
+            .dom('[data-test-dynamicScanModal-startBtn]')
+            .isNotDisabled()
+            .hasText('t:modalCard.dynamicScan.start:()');
+
+          await click('[data-test-dynamicScanModal-startBtn]');
+        }
+
+        const notify = this.owner.lookup('service:notifications');
+
+        assert.strictEqual(
+          notify.successMsg,
+          automatedScan
+            ? 't:scheduleDynamicscanSuccess:()'
+            : 't:startingScan:()'
+        );
+
+        // assert.strictEqual(
```

Comment:
Commented assertions?

Reply:
Here I am not able to make final assertion because poll service is not working properly in test environment so just keeping it for future reference